library/Director/Integration/BackendInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Icinga\Module\Director\Integration;
+
+use Icinga\Web\Url;
+
+interface BackendInterface
+{
+    /**
+     * Whether the backend is available
+     *
+     * @return bool
+     */
+    public function isAvailable(): bool;
```

Comment:
Since this will be exclusive to the monitoring backend, I'd suggest to remove this from the interface.